### PR TITLE
created Api_Update

### PR DESCRIPTION
**Pull request by AntsiferauGodel**

### CODE REVIEW

File: AreaMonitor.WebApi/AreaMonitor.WebApi/Controllers/RecordController.cs [hunk 1]
```diff
@@ -49,5 +49,12 @@ public void DeleteRecord([FromBody]Record record)
         {
             recordService.DeleteRecord(record);
         }
+
+        // /api/Record/UpdateRecord/{record}
+        [HttpPost]
+        public void UpdateRecord([FromBody]Record record)
```
Comment (pipalalala) [inline AreaMonitor.WebApi/AreaMonitor.WebApi/Controllers/RecordController.cs]: Please, add null parameter checking everywhere
Like this:
if (record == null)
{
    throw new ArgumentNullException(nameof(record));
}

File: AreaMonitor.WebApi/AreaMonitor.WebApi.BL/Services/RecordService.cs [hunk 2]
```diff
@@ -45,7 +45,7 @@ public IEnumerable<Record> GetRecordsForUser(string userId)
 
         public void UpdateRecord(Record record)
```
Comment (pipalalala) [inline AreaMonitor.WebApi/AreaMonitor.WebApi.BL/Services/RecordService.cs]: Please, add null parameter checking everywhere

File: AreaMonitor.WebApi/AreaMonitor.WebApi/Controllers/RecordController.cs (hunk 1 above)
Comment (AntsiferauGodel) [reply AreaMonitor.WebApi/AreaMonitor.WebApi/Controllers/RecordController.cs]: we will discus it later

File: AreaMonitor.WebApi/AreaMonitor.WebApi.BL/Services/RecordService.cs (hunk 2 above)
Comment (AntsiferauGodel) [reply AreaMonitor.WebApi/AreaMonitor.WebApi.BL/Services/RecordService.cs]: we will discus it later

**Pull request merged**